data/i18n/en/en.toml

```diff
@@ -545,6 +548,17 @@ other = "Cherry Pick Deadline"
 [release_end_of_life_date]
 other = "End Of Life Date"
 
+# Also see release_maintenance_and_end_of_life_details_past
+[release_maintenance_and_end_of_life_details_current]
+other = "Kubernetes {{ .minor_version }} enters maintenance mode on {{ .maintenance_mode_start_date }}; the End of Life date for Kubernetes {{ .minor_version }} is {{ .release_eol_date }}."
+
+# Used if the maintenance mode date is in the past, otherwise
+# see release_maintenance_and_end_of_life_details_current
+[release_maintenance_and_end_of_life_details_past]
+other = """🛈 **Kubernetes {{ .minor_version }} entered maintenance mode on {{ .maintenance_mode_start_date }}**.
```

Comment:
What's this character at the beginning of the sentence? It's showing as a box in GitHub preview, and doesn't show at all on the website.

<img width="536" alt="image" src="https://github.com/user-attachments/assets/276738ed-2294-46c1-9d5f-4a201e1729d5">

Reply:
Ah, that's similar to :information_source: 

I can switch it to something more widely supported.

Reply:
That would be great, other than that, the PR looks good to me!

Reply:
(done)